_includes/releases.html

```diff
@@ -1,4 +1,4 @@
-<table class="table table-responsive-lg table-dark downloads">
+<table id="{{include.table_id}}" class="table table-responsive-lg table-dark downloads" {% if include.releases.size == 0 %}style="display: none;"{% endif %}>
```

Comment:
So, the tables are hidden, but the headings are not, and there's no else to show the link to Github releases if both the JS API and Jekyll fail to get releases? Won't this look basically identical to https://media.discordapp.net/attachments/610531541889581068/1059327221019463750/image.png in that case, but without the table headings?